### PR TITLE
BZ#2131737 add insructions for clearing PKI files in DWH

### PR DESCRIPTION
**Pull request by emarcusRH**
DOWNSTREAM ONLY - this content is not available in upstream the oVirt documentation

see file preview:
https://github.com/oVirt/ovirt-site/blob/ea39ed92400c4ca25555824444e060329a52f253/source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc

https://github.com/oVirt/ovirt-site/blob/51ce16a62b079c1bfb2db974cdda4f1285ec7b27/source/documentation/common/database/proc-Migrating_the_Data_Warehouse_Database_to_a_Separate_Machine.adoc

Signed-off-by: emarcusRH <emarcus@redhat.com>

[ Bug fix]

Fixes issue # |\[BZ#2131737](https://bugzilla.redhat.com/show_bug.cgi?id=2131737)  
Changes proposed in this pull request:

- add insructions to DWH Configure procedure for clearing PKI files-

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/main/CONTRIBUTING.md): (please @emarcusRH )

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc [hunk 1]
```diff
@@ -16,7 +16,18 @@ The order of the options or settings shown in this section may differ depending
      -e '/^OVESETUP_CONFIG\/fqdn/d' \
      /etc/ovirt-engine-setup.conf.d/20-setup-ovirt-post.conf
 ----
-
+. Remove the apache/grafana PKI files, so that they are regenerated by `engine-setup`, if enabled, using correct values. 
+//If they are missing, that's ok:
```
Comment (apinnick) [inline source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc:20]: You might want to remove this comment unless it needs to be there.

File: source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc [hunk 2]
```diff
@@ -16,7 +16,18 @@ The order of the options or settings shown in this section may differ depending
...
++
+----
+# rm -f \
+        /etc/pki/ovirt-engine/certs/apache.cer \
```
Comment (apinnick) [inline source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc:24]: Change indents to 2 spaces.

File: source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc [hunk 3]
```diff
@@ -16,7 +16,18 @@ The order of the options or settings shown in this section may differ depending
      -e '/^OVESETUP_CONFIG\/fqdn/d' \
      /etc/ovirt-engine-setup.conf.d/20-setup-ovirt-post.conf
 ----
-
+. Remove the apache/grafana PKI files, so that they are regenerated by `engine-setup`, if enabled, using correct values.
```
Comment (apinnick) [inline source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc:19]: It's not clear what "if enabled" means in this context. Do you need this phrase?

**Review by apinnick — CHANGES_REQUESTED** [review]

File: source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc (hunk 1 above)
Comment (didib) [reply source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc]: Not sure why it's a comment. It's a note to the user - if 'rm' emits errors about some files missing, that's ok.

File: source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc (hunk 3 above)
Comment (didib) [reply source/documentation/common/database/proc-configuring-the-new-data-warehouse-machine.adoc]: 'enabled' means that the user replied 'yes' when asked whether to configure apache/grafana.

**Review by RichardHoch — APPROVED** [review]

**Pull request merged**